docs/Protocol/publishBlockStream.md

```diff
@@ -0,0 +1,164 @@
+# Block Node Connect Protocol for `publishBlockStream`
```

Comment:
We could add overview section, briefly describing what this document is all about. OR we can add a readme in the root folder itself, describing what protocol design documents are stored here.

Reply:
Good suggestion. I'll add both.